packages/nitro-protocol/gas-benchmarks/benchmark.test.ts

```diff
@@ -270,5 +274,7 @@ describe('Consumes the expected gas for sad-path exits', () => {
         (a, b) => a + b
       ) - gasRequiredTo.ETHexitSadVirtualFunded.vanillaNitro.total
     ).toEqual(gasRequiredTo.ETHexitSadVirtualFunded.vanillaNitro.total);
+
+    await assertETHSanityChecks({Alice: 5, Bob: 5, Ingrid: 0}, {LforG: 0, G: 0, J: 0, X: 0});
```

Comment:
I see the 5,5 outcome specified in the creation of channel `X` in fixtues.ts.

A future work could be to add a utility function to `TestChannel` which outputs the inputs for these sanity assertions, avoiding the duplicated input of this sort of information.

Reply:
5357799 and 902889f